### ISSUE TITLE
Update aria labels for online material toggle

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1770]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

collection or component page, e.g. https://findingaids.princeton.edu/catalog/C0178

### Description of error / warning

When navigating the page with VoiceOver
- when you get to the the online image shouldn't announce; it's just illustrative and its confusing when it tells you there's an image
- The text and label repeat each other.

### WCAG success criterion

[4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG22/#name-role-value)

"ARIA is used appropriately to enhance accessibility when HTML is not sufficient."

### WCAG conformance level

A

### Recommended fix

- give the image `alt=""`
- Figure out how to make the label only be read once

### Tool used to find the problem

Voiceover in Safari

### Screenshot

```html
<div class="toc-online-toggle al-online-material-icon">
          <span class="blacklight-icons blacklight-icon-online">
<svg height="24" width="24" viewBox="0 0 640 512" aria-label="Online" role="img">
...
</svg>
</span>
          <label for="tocOnlineToggle" class="online-toggle-label">
            <span> Only show material containing online material </span>
           <div class="toggle">
              <input type="checkbox" id="tocOnlineToggle">
              <span class="toc-slider"></span>
          </div>
          </label>
        </div>
```
